src/brevitas/export/common/handler/qcdq.py

```diff
@@ -122,7 +123,11 @@ def dequantize_symbolic_kwargs(cls, scale, zero_point, bit_width, is_signed):
             zero_point = zero_point.flatten()
         zp = to_0dim_if_scalar(zero_point)
         zp = zp.expand_as(scale)
-        zp = cls.zero_point_with_dtype(is_signed, bit_width, zp)
+
+        if isinstance(tensor_quant, FloatQuant):
```

Comment:
This check might break very easily.
Let's discuss how to get a similar functionality

Reply:
I'd expect it would probably break, i'm still experimenting with ways to get the necessary information for the export to fp8 without splitting the classes in the export hierarchy into floating point and integer equivalents. This method seems to work for the simple test case written but it hasn't been tested beyond that.